.docker/source/Dockerfile

```diff
@@ -4,35 +4,31 @@
 FROM moveit/moveit:melodic-ci-shadow-fixed
 MAINTAINER Dave Coleman dave@picknik.ai
 
-ENV CATKIN_WS=/root/ws_moveit
```

Comment:
No need for these settings. They are derived from `melodic-ci-shadow-fixed`.

Reply:
Dear @davetcoleman,
unfortunately I was too involved in preparing and hosting our Automatica booth. However, now I'm recovered from this and commented on your feedback. Sorry for the delay on this PR.